src/Typography/Font.php

```diff
@@ -11,6 +11,8 @@ class Font implements FontInterface
     protected float $size = 12;
     protected float $angle = 0;
     protected mixed $color = '000000';
+    protected mixed $strokeColor = 'red';
```

Comment:
```suggestion
    protected mixed $strokeColor = 'ffffff';
```

Text color default is `000000` so this is the opposite.